RadarSDK.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name                  = 'RadarSDK'
-  s.version               = '3.13.5'
+  s.version               = '3.14.0'
```

Comment:
@lmeier I recall us wanting to make this a major version update?

Reply:
Given that we're keeping existing behavior, we can keep this a minor. I expect that sometime this fall we'll publicize all these changes a bit more and roll into a `4.0.0`